src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -683,7 +683,8 @@
     <value>PowerToys Run</value>
   </data>
   <data name="About_PowerRename.Text" xml:space="preserve">
-    <value>About Power Rename</value>
+    <value>About PowerRename</value>
+    <comment>do not loc the product name</comment>
   </data>
   <data name="PowerRename_Image.AutomationProperties.Name" xml:space="preserve">
     <value>Power Rename</value>
```

Comment:
I wonder if I should change this as well?

Reply:
PowerRename_Image.AutomationProperties.Name

Reply:
Please

Reply:
Done!

Reply:
Can you add same comment about not loc'ing?

Reply:
Done!